ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -543,6 +546,40 @@ struct FrameworkBuilder {
               "\(error.localizedDescription)")
           }
         }
+        do {
+          // Frameworks built from only Swift sources will contain only two
+          // headers: the CocoaPods-generated umbrella header and the
+          // Swift-generated Swift header. If the framework's `Headers`
+          // directory contains more than two resources, then it is assumed
+          // that the framework was built from mixed language sources.
+          let headersDir = destination.appendingPathComponent("Headers")
+          let headers = try fileManager.contentsOfDirectory(
+            at: headersDir,
+            includingPropertiesForKeys: nil
+          )
+          if headers.count > 2 {
```

Comment:
These changes treat Sessions and Storage as mixed language pods which they technically are ([1](https://github.com/firebase/firebase-ios-sdk/blob/7502fe3d09c23a90938bdd18ca0c2b64d711ec32/FirebaseSessions.podspec#L37-L40), [2](https://github.com/firebase/firebase-ios-sdk/blob/7502fe3d09c23a90938bdd18ca0c2b64d711ec32/FirebaseStorage.podspec#L35-L38)). The resulting module maps for those products now contain the custom module map contents that was previously getting dropped. This looks fine to me, thoughts?

<details>
<summary>Manual diff results...</summary>

```
‣ diff -r -x '*.plist' -x '*.abi.json' -x '.DS_Store' -x '*.swiftinterface' -x '*.swiftmodule' -x '*.h' /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase | sed '/Binary\ files\ /d' 
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/ios-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/macos-arm64_x86_64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/macos-arm64_x86_64/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "AppKit"
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/tvos-arm64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/tvos-arm64/FirebaseSessions.framework/Modules/module.modulemap
2,9c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
10a7
> 
14c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/tvos-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseCrashlytics/FirebaseSessions.xcframework/tvos-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap
2,9c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
10a7
> 
14c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64_x86_64-simulator/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/ios-arm64_x86_64-simulator/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/macos-arm64_x86_64/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/macos-arm64_x86_64/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/tvos-arm64/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/tvos-arm64/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/tvos-arm64_x86_64-simulator/FirebaseFirestore.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseFirestore/FirebaseFirestore.xcframework/tvos-arm64_x86_64-simulator/FirebaseFirestore.framework/Modules/module.modulemap
12,15d11
< module FirebaseFirestore.Swift {
<   header "FirebaseFirestore-Swift.h"
<   requires objc
< }
\ No newline at end of file
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/ios-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/macos-arm64_x86_64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/macos-arm64_x86_64/FirebaseSessions.framework/Modules/module.modulemap
2,10c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "AppKit"
<   link framework "CoreTelephony"
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
11a7
> 
15c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/tvos-arm64/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/tvos-arm64/FirebaseSessions.framework/Modules/module.modulemap
2,9c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
10a7
> 
14c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/tvos-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebasePerformance/FirebaseSessions.xcframework/tvos-arm64_x86_64-simulator/FirebaseSessions.framework/Modules/module.modulemap
2,9c2,5
< umbrella header "FirebaseSessions-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "SystemConfiguration"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseSessions-umbrella.h"
> 
>   export *
>   module * { export * }
10a7
> 
14c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64_x86_64-maccatalyst/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64_x86_64-simulator/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/ios-arm64_x86_64-simulator/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/macos-arm64_x86_64/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/macos-arm64_x86_64/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "AppKit"
<   link framework "Foundation"
<   link framework "Security"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/tvos-arm64/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/tvos-arm64/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
diff --color -r -x *.plist -x *.abi.json -x .DS_Store -x *.swiftinterface -x *.swiftmodule -x *.h /Users/nickcooke/Downloads/zip-build-post-change/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/tvos-arm64_x86_64-simulator/FirebaseStorage.framework/Modules/module.modulemap /Users/nickcooke/Downloads/zip-build-master-HEAD/10_16_0/Firebase/FirebaseStorage/FirebaseStorage.xcframework/tvos-arm64_x86_64-simulator/FirebaseStorage.framework/Modules/module.modulemap
2,8c2,5
< umbrella header "FirebaseStorage-umbrella.h"
< export *
< module * { export * }
<   link framework "Foundation"
<   link framework "Security"
<   link framework "UIKit"
<   link "z"
---
>   umbrella header "FirebaseStorage-umbrella.h"
> 
>   export *
>   module * { export * }
9a7
> 
13c11
< }
\ No newline at end of file
---
> }
```
</details>

Reply:
Just to confirm, the "<" indicates the new version. If so, 👍

Reply:
@paulb777 yes, thanks for looking

Reply:
W.r.t. to the approach I'm using to determine if the SDK is mixed language, this is why I went with this way: 

- We could make this check explicit by forcing us to manually specify whether a pod is mixed language in the `FirebaseManifest.swift`, but that becomes piece of documentation we need to maintain and think about in the future. So I opted against that.
- Ideally, we can determine whether an SDK is mixed language by storing such info in the `Pod` data model. I opted against this for two reasons, doing it implicitly would be a little tricky there isn't a guaranteed way to know if a pod contains mixed sources until it is cloned and the workspace is created and the sources are scanned. This did not seem straight forward so I looked for other ways that would hint a pod is mixed language (which led me to the proposed approach of counting headers).